Templates/Integrations/Authentication/Authentication.py

```diff
@@ -0,0 +1,410 @@
+    """Unlocks a vault by vault ID.
+    """
+    vault_to_lock = args.get('vault_id', '')
+    raw_response = client.unlock_vault_request(vault_to_lock)
+    vaults = raw_response.get('vault')
+    if vaults and vaults[0].get('vaultId') and vaults[0].get('isLocked') is False:
+        vault_object = vaults[0]
+        title = f'{INTEGRATION_NAME} - Vault {vault_to_lock} has been unlocked'
+        context_entry = build_vaults_context(vault_object)
+        context = {f'{INTEGRATION_CONTEXT_NAME}.Vault(val.ID && val.ID === obj.ID)': context_entry}
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        raise DemistoException(f'{INTEGRATION_NAME} - Could not unlock vault ID: {vault_to_lock}')
+
+
+def list_vaults(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Lists all vaults.
+    """
+    max_results = int(args.get('max_results', 0))
+    raw_response = client.list_vaults_request(max_results)
+    vaults = raw_response.get('vault')
+    if vaults:
+        title = f'{INTEGRATION_NAME} - Total of {len(vaults)} has been found.'
+        context_entry = build_vaults_context(vaults)
+        context = {f'{INTEGRATION_CONTEXT_NAME}.Vault(val.ID && val.ID === obj.ID)': context_entry}
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No vaults found.', {}, {}
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def main():  # pragma: no cover
+    params = demisto.params()
+    base_url = params.get('url', '').rstrip('/') + '/api/v1'
+    verify = not params.get('insecure', False)
+    proxy = params.get('proxy') == 'true'
+    client = Client(base_url, verify=verify, proxy=proxy)
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+
+    # Switch case
+    commands = {
+        'test-module': test_module,
+        'fetch-credentials': fetch_credentials,
+        f'{INTEGRATION_COMMAND_NAME}-list-accounts': list_accounts,
+        f'{INTEGRATION_COMMAND_NAME}-lock-account': lock_account,
+        f'{INTEGRATION_COMMAND_NAME}-unlock-account': unlock_account,
+        f'{INTEGRATION_COMMAND_NAME}-reset-account': reset_account,
+        f'{INTEGRATION_COMMAND_NAME}-lock-vault': lock_vault,
+        f'{INTEGRATION_COMMAND_NAME}-unlock-vault': unlock_vault,
+        f'{INTEGRATION_COMMAND_NAME}-list-vaults': list_vaults
+    }
+    try:
+        if command == 'fetch-credentials':
+            # Fetch credentials is handled, no return statement.
+            commands[command](client)
+        if command in commands:
+            return_outputs(*commands[command](client, demisto.args()))
+    # Log exceptions
+    except Exception as e:
+        err_msg = f'Error in {INTEGRATION_NAME} - [{e}]'
+        return_error(err_msg, error=e)
+
+
+if __name__ == '__builtin__':  # pragma: no cover
```

Comment:
whats the pragma comment here for ?

Reply:
for coverage that I run on the files.
I'm trying to add it to the circle also.